tgui/packages/tgui/styles/interfaces/ChemHeater.scss

```diff
@@ -0,0 +1,66 @@
+@use '../base.scss';
+@use '../colors.scss';
+
+$neutral: #AAAAAA !default;
+$hot: #f2711c !default;
+$cold: #2185d0 !default;
+
+@keyframes heating {
+  to {
+    box-shadow: 0 0 15px 0 $hot;
+    border-color: $hot;
+  }
+}
+
+@keyframes cooling {
+  to {
+    box-shadow: 0 0 15px 0 $cold;
+    border-color: $cold;
+  }
+}
+
+@keyframes heating_fill {
+  to {
+    color: $hot;
+  }
+}
+
+@keyframes cooling_fill {
+  to {
+    color: $cold;
+  }
+}
+
+.TemperatureNumber {
+  transition: color 500ms ease-out;
+}
+
+.TemperatureArrow {
```

Comment:
Should be scoped to the relevant interface, or made into a common component and then put in a file as such.